lib/improver/ensemble_calibration/ensemble_calibration.py

```diff
@@ -545,11 +574,11 @@ def compute_initial_guess(
 
         if (predictor_of_mean_flag.lower() == "mean" and
                 not estimate_coefficients_from_linear_model_flag):
-            initial_guess = [1, 1, 0, 1]
+            initial_guess = [0, 1, 0, 1]
         elif (predictor_of_mean_flag.lower() == "realizations" and
               not estimate_coefficients_from_linear_model_flag):
-            initial_guess = [1, 1, 0] + np.repeat(
-                1, no_of_realizations).tolist()
+            initial_guess = [0, 1, 0] + np.repeat(
+                np.sqrt(1. / no_of_realizations), no_of_realizations).tolist()
```

Comment:
For an ensemble with 3 realizations, the change to the last coefficient is from 1 to np.sqrt(1./3). The square root is only included as the coefficient is squared within the minimisation later, but the main point is that the coefficient can be interpreted as each realization having a weight of 1/3 each i.e. each of the ensemble members have an equal weight. This is better than the previous initial guess of 1, as although this was also saying that the ensemble realizations had equal weight, the minimisation was having to do more work to get coefficients that effectively added up to 1 (i.e. 1/3 + 1/3 +1/3), rather than 3 (1 + 1 +1), as the total weight for all realizations would be expected to add up to 1, rather than 3. This meant that previously, the minimisation was spending time forcing down these coefficients to be more like ~1/3.